table/get_context.cc

```diff
@@ -241,6 +245,8 @@ bool GetContext::SaveValue(const ParsedInternalKey& parsed_key,
     }
 
     size_t ts_sz = ucmp_->timestamp_size();
+    Slice ts = ExtractTimestampFromUserKey(parsed_key.user_key, ts_sz);
```

Comment:
We can save this function call for `ts_sz=0` cases, if we just declare ts here, which defaults it to an empty Slice. And only execute `ExtractTimestampFromuserKey` in the code block below.